src/frontend/src/lib/derived/network.derived.ts

```diff
@@ -25,6 +25,10 @@ export const networkICP: Readable<boolean> = derived([networkId], ([$networkId])
 	isNetworkIdICP($networkId)
 );
 
+export const networkBitcoin: Readable<boolean> = derived([networkId], ([$networkId]) =>
+	isNetworkIdBitcoin($networkId)
```

Comment:
maybe should we prevent this to be enabled if `VITE_BITCOIN_MAINNET` is not true for now?

@AntonioVentilii-DFINITY, since you introduced the env variable, wdyt?

Reply:
theoretically, any network that could be submitted to this function comes from a place that already filters using that flag (for example `enabledBitcoinTokens` or `networks` stores).

No strong opinion to add it here: it would just be another layer of insurance, but I am almost sure it is not needed.

@lmuntaner you decide

Reply:
If already filtered I think we do not have to duplicate the flag. Thanks for the feedback Antonio.